webapi/Controllers/ChatHistoryController.cs

```diff
@@ -260,4 +268,108 @@ public async Task<IActionResult> GetAllChatSessionsAsync(string userId)
 
         return this.NotFound($"No chat session found for chat id '{chatId}'.");
     }
+
+    /// <summary>
+    /// Delete a chat session.
+    /// </summary>
+    /// <param name="userId">Unique Id of user who initiated the request.</param>
+    /// <param name="sessionId">The chat id.</param>
+    [HttpDelete]
+    [Route("chatSession/{sessionId:guid}")]
```

Comment:
I feel like other places we have used 1chatId` instead of `sessionId`. Can we be consistent with our naming?

Reply:
Renamed to chatId